### PR TITLE
Feature/hab 15 snapkit integration

### PR DESCRIPTION
**Pull request by VikkiRy**

### CODE REVIEW

**Comment by nikGayko** [comment]
Don't forget to change target branch to dev

**Comment by VikkiRy** [comment]
Target branch changed

**Review by HKurylchykava — APPROVED** [review]

File: BadHabbit/.swiftlint.yml [hunk 1]
```diff
@@ -4,6 +4,7 @@ excluded: # paths to ignore during linting. Takes precedence over `included`.
   
 disabled_rules:
 - trailing_whitespace
+- blanket_disable_command
```
Comment (nikGayko) [inline BadHabbit/.swiftlint.yml]: Could you tell what this rule check ?

File: BadHabbit/BadHabbit/XCAssets+Generated.swift [hunk 2]
```diff
@@ -0,0 +1,105 @@
+// swiftlint:disable all
+// Generated using SwiftGen — https://github.com/SwiftGen/SwiftGen
+
+#if os(macOS)
```
Comment (nikGayko) [inline BadHabbit/BadHabbit/XCAssets+Generated.swift]: Why this file added here ?

Comment (VikkiRy) [reply BadHabbit/BadHabbit/XCAssets+Generated.swift]: File removed

**Review by nikGayko — APPROVED** [review]

**Pull request merged**